ex_topo/aws_sg_topology.yml

```diff
@@ -0,0 +1,30 @@
+---
+topology_name: "aws_sg_test_topology"
+resource_groups:
+  -
+    resource_group_name: "awssgtest"
+    res_group_type: "aws"
+    res_defs:
+       -
+         res_type: "aws_sg"
+         res_name: "aws_test_sg"
+         description: "AWS Security Group with ssh access"
+         region: "us-east-1"
+         rules:
```

Comment:
Let's use the following rules here as I believe they cover the most common use case (inbound ping allowed, inbound SSH allowed, outbound access allowed:
```
    rules:
      - proto: icmp
        to_port: -1
        from_port: 8 # type 8 is ICMP echo request
        cidr_ip: 0.0.0.0/0
      - proto: tcp
        type: ssh
        to_port: 22
        from_port: 22
        cidr_ip: 0.0.0.0/0
    rules_egress:
      - proto: all
        type: all
        cidr_ip: 0.0.0.0/0
```